manifest.php

```diff
@@ -43,11 +43,11 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '25.4.2',
+    'version'     => '25.4.3',
     'author'      => 'Open Assessment Technologies',
     'requires' => [
         'taoItems' => '>=10.8.2',
-        'tao'      => '>=43.1.0',
+        'tao'      => '>=45.1.3',
```

Comment:
Just a friendly reminder, in case you decide to accept [that suggestion](https://github.com/oat-sa/tao-core/pull/2621#discussion_r462253756).

Don't forget to update the dependency version here as well.